packages/document/module-doc/docs/en/api/config/build-config.md

```diff
@@ -703,7 +713,7 @@ style_inject_es_default(css_248z);
 With `inject` turned on, you need to pay attention to the following points.
 
 - `@import` in the css file will not be processed. So if you have `@import` in your css file, then you need to bring in the css file manually in js (less,scss files don't need it because they will be preprocessed).
-- The impact of `sideEffects` needs to be considered, by default our builder will consider it as a side effect, if the `sideEffects` field set in your project or in the package.json of the three-way package and it does not contain this css file, then you will get a warning
+- The impact of `sideEffects` needs to be considered, by default our builder will consider it as a side effect, if the `sideEffects` field set in your project or in the package.json of the three-party package and it does not contain this css file, then you will get a warning
```

Comment:
```suggestion
- The impact of `sideEffects` needs to be considered, by default our builder will consider it as a side effect, if the `sideEffects` field set in your project or in the package.json of the third party package and it does not contain this css file, then you will get a warning
```

Reply:
so,  third party or three-party?

Reply:
Fine, i' ll replace all three-party with third-party

Reply:
Yes, the correct term is "third party". "three party" is not a commonly used term in English.